src/ts/Conviva.ts

```diff
@@ -0,0 +1,205 @@
+export declare namespace Conviva {
```

Comment:
Declarations should be in a separate file

Reply:
What do you mean by that, since this is a separate file?

Reply:
Sorry I thought you moved it into `ConvivaAnalytics.ts` because I assumed it would be a simple file rename otherwise.

Reply:
No worries, will revert it to `.d.ts`, though.